src/components/buttons/styles.module.css

```diff
@@ -1,7 +1,11 @@
 .button {
   display: inline-flex;
+  gap: 0.5rem;
   align-items: center;
   justify-content: center;
+  align-self: center;
```

Comment:
Acho que a gente nao rpecisa desse align self

Reply:
Esse align-self veio pra não dependermos da regra no pai, naquele artigo que compartilhasse com a gente:
https://ishadeed.com/article/styling-the-good-old-button/#anchor-link-a-vs-button

<img width="627" alt="Screenshot 2024-11-12 at 20 30 35" src="https://github.com/user-attachments/assets/3668f914-e6b3-42c3-9a83-576af248d93a">

Reply:
Ah boa!